Blacklist Indicators with Crowdstrike Falcon from Slack (#354)
rapid7/insightconnect-workflows: workflows/Blacklist_Indicators_with_CrowdStrike_Falcon_from_Slack/help.md

# Description

> Use the simplicity of a chat command in Slack to block an indicator such as IPv6, IPv4, MD5 hash, SHA256 and domain in your Crowdstrike Falcon.

Reviewer: ```suggestion
Use the simplicity of a chat command in Slack to block an indicator such as IPv4 and IPv6 address, MD5 and SHA256 hashes, and domains in CrowdStrike Falcon.
```

Once the workflow has been imported, **Update the first step with the channel name to suit your Slack environment!** by editing the input with the preset text of `change_me` to match the channel to monitor.
And then **Edit Each CrowdStrike Falcon step to specify the group you are using** In these steps, specify the group you would like to add and remove host objects from.

After import, activate the workflow in order to trigger it.

### Usage

*This workflow will trigger in any direct messages to your Chatbot **or** any message in a channel directed @ your Chatbot. Note the Chatbot must be in the channel in order to trigger the workflow this way.*

For example:
* `@Rapid7 InsightConnect blacklist-indicators 198.51.100.100`

The workflow will reply when it completes.

## Technical Details

Plugins utilized by workflow:

|Plugin|Version|Count|
|----|----|--------|
|CrowdStrike Falcon|2.2.0|1|
|Type Converter|1.6.0|1|

## Troubleshooting

_There is no troubleshooting information at this time_

# Version History

* 1.0.0 - Initial workflow

# Links

## References

> * [Crowdstrike](https://www.crowdstrike.com/)

Reviewer: ```suggestion
* [CrowdStrike](https://www.crowdstrike.com/)
```

> * [Crowdstrike Falcon API](https://falcon.crowdstrike.com/support/documentation)

Reviewer: ```suggestion
* [CrowdStrike Falcon API](https://falcon.crowdstrike.com/support/documentation)
```
